getting_started/step_by_step/resources.rst

```diff
@@ -6,106 +6,97 @@ Resources
 Nodes and resources
 -------------------
 
-So far, :ref:`Nodes <class_Node>`
-have been the most important datatype in Godot as most of the behaviors
-and features of the engine are implemented through them. There is
-another datatype that is equally important:
+Up to this tutorial, we focused on the :ref:`Nodes <class_Node>`
+class in Godot as that's the one you use to code behavior and
+and most of the engine's features rely on them. There is
+another datatype that is just as important:
 :ref:`Resource <class_Resource>`.
 
-Where *Nodes* focus on behaviors, such as drawing a sprite, drawing a
-3D model, physics, GUI controls, etc,
-
-**Resources** are mere **data containers**. This means that they don't
-do any action nor process any information. Resources just contain
-data.
-
-Examples of resources are
-:ref:`Texture <class_Texture>`,
-:ref:`Script <class_Script>`,
-:ref:`Mesh <class_Mesh>`,
-:ref:`Animation <class_Animation>`,
-:ref:`AudioStream <class_AudioStream>`,
-:ref:`Font <class_Font>`,
-:ref:`Translation <class_Translation>`,
-etc.
-
-When Godot saves or loads (from disk) a scene (.tscn or .scn), an image
-(png, jpg), a script (.gd) or pretty much anything, that file is
```

Comment:
(png, jpg) -> (.png, .jpg)

Reply:
This is a removed line, it's in red as you can see